fdbrpc/sim2.actor.cpp

```diff
@@ -2132,11 +2142,16 @@ class Sim2 final : public ISimulator, public INetworkConnections {
 	bool yielded;
 	int yield_limit; // how many more times yield may return false before next returning true
 
+private:
+	static MockDNS mockDNS;
```

Comment:
Remove static. There is only one sim2 per simulation to simulate an environment.

Reply:
Makes sense.